.gitignore

```diff
@@ -1,3 +1,14 @@
 *.pdf
 !example/businesscard.pdf
 node_modules
+
+# swap
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+# session
+Session.vim
+# temporary
+.netrwhist
+*~
+# auto-generated tag files
+tags
```

Comment:
Please only add project-specific files to a gitignore.
IMO everything else belongs into a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)